src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/SubAtlasCommandScriptTemplate.groovy

```diff
@@ -0,0 +1,7 @@
+import java.util.function.*;
+import org.openstreetmap.atlas.geography.atlas.items.*;
+Predicate<AtlasEntity> predicate = {
+    %s;
+};
+
+return predicate;
```

Comment:
You may be able to do without any string manipulation.

Reply:
Ah yes this is a good idea. Let's not merge until implemented.